src/Services/BookingService/BookingService.cs

```diff
@@ -177,7 +187,13 @@ public async Task ProcessMonthRequest(Dictionary<string, object> viewModel, stri
 
             var bookingSearchResultsKey = $"{bookingElement.Properties.QuestionId}{BookingConstants.APPOINTMENT_TYPE_SEARCH_RESULTS}";
 
+            var bookingInformationCacheKey = $"{bookingElement.Properties.QuestionId}{BookingConstants.APPOINTMENT_TYPE_SEARCH_RESULTS}";
```

Comment:
bookingInformationCacheKey is the same value at bookingSearchResultsKey above.

Reply:
Removed. Didn't add it in the first place, but it's removed.